owl/EASE-OBJ.owl

```diff
@@ -438,11 +595,65 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#AffectedObject -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#AffectedObject">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Role"/>
+        <rdfs:comment>An object that undergoes the event.</rdfs:comment>
+    </owl:Class>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Alteration -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Alteration">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#Disposition"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#modifies"/>
+                <owl:someValuesFrom rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Quality"/>
```

Comment:
Is it some or is it only?

I can have a spraycan of paint. Clearly it has the disposition to paint a surface. However, I could store this spraycan, unused, until the end of time, and not have it participate in any painting event.

Qualities, also, should not be treated as abstract disembodied individuals. DUL's axioms are a bit weaker here in that they don't require a Quality to be "is quality of" something, but the DUL comment about qualities is clear that a Quality is "Any aspect of an Entity (but not a part of it), which cannot exist without that Entity."

So a spraycan of paint can, indeed, modify colors, but when we have in mind a Color individual we should also have in mind the Object (or whatever) individual that Color belongs to. So there cannot be an abstract statement that this spraycan has a disposition to modify some platonic Color individual. It can, in a particular event, be used to modify the color of a particular wall.

Reply:
only, you are right